src/overlays/ovl_i4/fox_fo.c

```diff
@@ -1,29 +1,1513 @@
+                    player->pos.y = 0.0f;
+                    player->unk_0E4 = 0.0f;
+                    player->unk_0EC = 0.0f;
+                    player->unk_0D0 = 0.0f;
+                    player->unk_114 = 0.0f;
+                    player->unk_0E8 = 180.0f;
+                    func_i4_8018906C();
+                } else {
+                    player->pos.x = 0.0f;
+                    player->unk_0E4 = 0.0f;
+                    player->unk_0EC = 0.0f;
+                    player->unk_114 = 0.0f;
+                    player->unk_0D0 = 0.0f;
+                    player->camAt.y = 16.0f;
+                    player->pos.y = -100.0f;
+                    player->unk_0E8 = 180.0f;
+                }
+                player->pos.z = -10000.0f;
+                gCsFrameCount = 0;
+                player->wings.unk_2C = 1;
+                player->unk_204 = 1;
+                func_800182F4(0x106400FF);
+                func_800182F4(0x116400FF);
+                func_8001C8B8(0);
+                if (D_80177930 == 0) {
+                    func_i4_801890EC(gActors, 3);
+                }
+            }
+            break;
+
+        case 10:
+            player->unk_234 = 1;
+            D_80178358 = 0;
+            D_8017835C = 4;
+            player->camEye.x = 400.0f;
+            player->camEye.y = 0;
+            player->camEye.z = 0.0f;
+            player->camAt.x = actorPtr[0].obj.pos.x;
+            player->camAt.y = actorPtr[0].obj.pos.y;
+            player->camAt.z = actorPtr[0].obj.pos.z;
+
+            if (gCsFrameCount == 100) {
+                player->unk_0D0 = 30.0f;
+                if (gTeamShields[1] > 0) {
+                    func_i4_801890EC(&gActors[3], 0);
+                }
+                if (gTeamShields[2] > 0) {
+                    func_i4_801890EC(&gActors[1], 1);
+                }
+                if (gTeamShields[3] > 0) {
+                    func_i4_801890EC(&gActors[2], 2);
+                }
+            }
+
+            if (gCsFrameCount == 496) {
+                player->unk_1D0 = 11;
+                D_80177A48[0] = 0.01f;
+                D_80177A48[1] = 0.0f;
+                D_80177A48[2] = -400.0f;
+                D_80177A48[3] = 0.0f;
+                actorPtr[3].iwork[14] = 2;
+                actorPtr[1].iwork[14] = 3;
+                actorPtr[2].iwork[14] = 4;
+            }
+
+            if (gCsFrameCount == 200) {
+                if (D_80177930 == 0) {
+                    func_8001D444(0U, 0x31U, 0U, 0xFFU);
+                } else {
+                    func_8001D444(0U, 0x26U, 0U, 0xFFU);
+                }
+            }
+
+            if (gCsFrameCount == 420) {
+                D_80177840 = 0x64;
+            }
+            break;
+
+        case 11:
+            D_80177A48[1] += D_80177A48[3];
+            Matrix_RotateX(gCalcMatrix, -0.17453292f, 0);
+            Matrix_RotateY(gCalcMatrix, D_80177A48[1] * 0.017453292f, 1);
+            src.y = 0.0f;
+            src.x = 0;
```

Comment:
I think when i messed around with this earlier, you could instead have
```suggestion
            src.x = 0.0f;
            src.y = 0;
```
which seems somewhat more real